spec/imsc-hrm.html

```diff
@@ -200,7 +200,19 @@
   </section>
 
   <section id='sotd'>
-   
+    <p>For this specification to exit the CR stage, the following independent factors of verifications MUST be demonstrated:</p>
+      <ul>
+        <li>at least two content producing implementations</li>
+        <li>at least one validating implementation</li></p>
```

Comment:
These do not conflict with the current draft Charter text, which is good.

In case a 2nd validator implementation arises, I would like to be able to exit CR on the basis of 2 validators and one content producing implementation. What would you think about rewording it to say at least 1 of each, AND at least 2 of one of them?

Reply:
Ok. I think we should aim for two content sources. How do we say that while still allowing the scenario you suggest?

Reply:
If you think 2 content sources is a minimum, we can leave it as it is. If you think 1 content + 2 validators is also adequate, then we should say "at least one" for both, and "at least three" in total.

Reply:
New wording works for me, thanks.